Source/SantaGUI/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@build_bazel_rules_apple//apple:macos.bzl", "macos_application")
+
+objc_library(
+    name = "SantaGUI_lib",
+    srcs = [
+        "SNTAboutWindowController.h",
+        "SNTAboutWindowController.m",
+        "SNTAccessibleTextField.h",
+        "SNTAccessibleTextField.m",
+        "SNTAppDelegate.h",
+        "SNTAppDelegate.m",
+        "SNTMessageWindow.h",
+        "SNTMessageWindow.m",
+        "SNTMessageWindowController.h",
+        "SNTMessageWindowController.m",
+        "SNTNotificationManager.h",
+        "SNTNotificationManager.m",
+        "main.m",
+    ],
+    includes = [
+        "../common",
+    ],
+    sdk_frameworks = [
+        "IOKit",
+        "SecurityInterface",
+    ],
+    xibs = [
+        "Resources/AboutWindow.xib",
+        "Resources/MessageWindow.xib",
+    ],
+    deps = [
+        "//Source/common:SNTBlockMessage",
+        "//Source/common:SNTConfigurator",
+        "//Source/common:SNTXPCControlInterface",
+        "//Source/common:SNTXPCNotifierInterface",
+        "@MOLCodesignChecker//:MOLCodesignChecker",
```

Comment:
nit: @MOLCodesignChecker
Maybe just run buildifier on all the BUILD files?

Reply:
Done